src/core/jsonschema/official_resolver.in.cc

```diff
@@ -106,16 +105,6 @@ auto sourcemeta::core::schema_official_resolver(std::string_view identifier)
              "https://json-schema.org/draft/2019-09/output/hyper-schema") {
     return sourcemeta::core::parse_json(
         R"EOF(@METASCHEMA_HYPERSCHEMA_2019_09_OUTPUT@)EOF");
-
-    // Just for compatibility given that this is such a common issue
```

Comment:
Are these changes related to ClangTidy? If not, can we have them as separate PRs?

Reply:
These are reported by clang-tidy